.github/workflows/qa.yml

```diff
@@ -102,7 +102,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       sonarqube:
-        image: sonarqube:lts-community
+        image: sonarqube@sha256:a32ae80ca6b62852a4d170a711f9a4b4aeac9a0c379cefb4b16de1df0fc10641
```

Comment:
Since this is an image we own, and we are willing to test on the latest LTS, I think it is better to keep `lts-latest`

Reply:
Thanks, the change is dropped.